src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -350,6 +349,14 @@ ClientRequest::process_op(
     }
   }
 
+  /**
+  * The previous stage of recover_missing is a concurrent phase.
+  * Checking for already_complete requests must done exclusively.
+  * Since get_obc is also an exclusive stage, we can merge both stages into
+  * a single stage and avoid stage switching overhead.
```

Comment:
nit: indentation space missing

Reply:
Done, sorry, I didn't notice this indentation issue.